source/jormungandr/tests/instant_system_new_default_routing_tests.py

```diff
@@ -292,6 +292,33 @@ def test_start_ridesharing_with_pt(self):
         assert 'ridesharing' in journeys[0].get('tags')
         assert journeys[0].get('durations').get('ridesharing') == 37
 
+        q = journey_basic_query + \
+            ("&first_section_mode[]=ridesharing"
+             "&first_section_mode[]=car"
+             "&last_section_mode[]=walking"
+             "&ridesharing_speed=2.5")
```

Comment:
If you add `car_speed=2.5` you should be able to test that you get a fourth journey car+PT
Also would be nice to test section types on each journey (and a comment to explain direct car, fallback ridesharing, etc.)